### DISCUSSION TITLE
AtroCore  modules

### DISCUSSION THREAD

**Discussion by ahmed26399** [atrocore/atrocore#496] [Q&A]
Hello,
I'am trying to install AtroCore on Xampp ( windows)
I installed AtroDAM in order to install AtroCore ( i notice that wasn't the same menu like demo version menu ) . I can't uninstall DAM because the crontab.
I installed AtroPim but i notice that the menu was the same like demo version menu.

Could u help me to know what is the module  installed in core  in order to test it because i can't installed now on a virtual machine.

Thanks in advance.

**Comment by atrolex** [comment]
Hi, AtroCore is the platform, it is always installed. Depending on the needs you can install PIM and/or DAM. Both are technically modules for AtroCore.

**Reply by ahmed26399** [reply]
Hello,
Thanks for your response.
Maybe my question wasn't clear. I need to install only an MDM modules, so i want only MDM modules from AtroCore to test it.
I m trying to install only the core module ( because it cover only MDM modules). I can' t unistall the DAM module from AtroDam ( because the cron problem and i m working on windows)
Please can you help me by  giving me names from this list included in the MDM module ( librairies, asset category, assets, Accounts, Streams) in order to test only the MDM modules 

Thanks in advance.

**Reply by atrolex** [reply]
just uninstall the DAM module, mark it to uninstall and click on the update button. I don't understand, what you mean by "cron problem". You don't have cron jobs running? If so, the system won't be usable. Please install it under linux, if you wish, in virtualbox for Windows.
